cdap-app-fabric/src/main/java/co/cask/cdap/app/preview/PreviewModule.java

```diff
@@ -81,5 +83,8 @@ protected void configure() {
     bind(PreviewManager.class).to(DefaultPreviewManager.class).in(Scopes.SINGLETON);
 
     bind(Scheduler.class).to(NoopScheduler.class);
+
+    bind(PreviewStore.class).to(DefaultPreviewStore.class).in(Scopes.SINGLETON);
+    expose(PreviewStore.class);
```

Comment:
Why need to expose? Which class that is not binded in this module is using `PreviewStore`?